index.html

```diff
@@ -18,11 +18,91 @@
+                  <img src="img\photo.jpg" class="img-fluid img-thumbnail rounded mx-auto d-block" alt="Responsive image">
+                  <p class="lead text-center">This is the place where you get to know more about me. I have a background in Management and now I am foraying into the world of programming!</p>
+                </div>
+              </div>
       </div>
-    </div><!-- container -->
+  </div> <!-- container1 -->
+
+    <div class="container">
+        <div class="row table-row no-gutters">
+            <div class="col-sm-12 col-lg-12 ">  
+            <h4 class="text-center font-weight-bold title">Work History</h4>
+            <table class="table-dark table-bordered table-striped">
+            <thead>
+                <tr>
+                    <th scope="col" class="text-center">Company</th>
+                    <th scope="col" class="text-center">Achievements</th>
+                    <th scope="col" class="text-center">Time Period</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td class="text-center">Company 1</td>
+                    <td class="text-center">I did almost nothing for company one in the 12 years I was there</td>
+                    <td class="text-center">1988-2000</td>
+                  </tr>
+                  <tr>
+                    <td class="text-center">Company2</td>
+                    <td class="text-center">I did lots of things for company two. it was pretty sweet</td>
+                    <td class="text-center">2000-2001</td>
+                  </tr>
+                  <tr>
+                    <td class="text-center">Company 3</td>
+                    <td class="text-center">I did so much for compnay 3 that I don't even know where to begin. Maybe I should start with Maecenas tincidunt semper tortor, at efficitur urna elementum in. In hac habitasse platea dictumst. </td>
+                    <td class="text-center">2001-2018</td>
+                  </tr>
+                </tbody>
+            </table>
+          </div> <!--column-->
+        </div>
+        <div class="row skills-row justify-content-center">
+          <div class="col-sm-3 col-lg-4">
+              <ul class="list-group">
+                    <li class="list-group-item list-group-item-dark text-center font-weight-bold">Skills</li>
+                    <li class="list-group-item list-group-item-dark text-center">HTML</li>
+                    <li class="list-group-item list-group-item-dark text-center">CSS</li>
+                    <li class="list-group-item list-group-item-dark text-center">Less</li>
+                    <li class="list-group-item list-group-item-dark text-center">Bootstrap</li>
+                  </ul>
+            </div> <!--column-->
+          </div> <!-- row-->
+      </div> <!-- container2 -->
+
+    <div class="container">
+      <div class="row contact-row justify-content-end">
+        <div class="col-sm-6 col-lg-3">   
```

Comment:
this may have something to do with the contact box going out of the area you want it in.  seems to only be at certain points.